src/game.cpp

```diff
@@ -3915,6 +3916,7 @@ void game::load( const save_t &name )
 
     // Now load up the master game data; factions (and more?)
     load_master();
+    load_external_options();
```

Comment:
Seems this function should be part of `load_master` as it's *always* required to be called after it. Also: it does not actually "load" anything, it updates cache values from already loaded data.